pkg/registry/storage/csinode/strategy_test.go

```diff
@@ -36,6 +37,7 @@ func getValidCSINode(name string) *storage.CSINode {
 					Name:         "valid-driver-name",
 					NodeID:       "valid-node",
 					TopologyKeys: []string{"company.com/zone1", "company.com/zone2"},
+					Allocatable:  &storage.VolumeNodeResources{Count: utilpointer.Int32Ptr(10)},
```

Comment:
a few more test cases for prepareForUpdate:

Feature gate off:
* allowed - Old object: allocatable not set; New object: allocable not set
* dropped - Old object: allocatable not set; New object: allocable set

Feature gate on:
* allowed - Old object: allocatable not set; New object: allocable not set
* allowed - Old object: allocatable not set; New object: allocable set

Reply:
Added